cfgov/v1/atomic_elements/organisms.py

```diff
@@ -31,6 +26,23 @@
 from v1.util import ref
 
 
+if wagtail.VERSION >= (2, 0):
```

Comment:
We've used `try`/`except` in all the other non-migrations (as far as I can tell), could we do that here for consistency?